### PR TITLE
Improved Docs

### PR DESCRIPTION
**Pull request by vishalvishw10**
I have corrected some spelling errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 * Your provider blocking info
 * OSS :)
 
 # Installation
 
-Debian, Ubuntu and Archlinux packages are supported!
+Debian, Ubuntu, and Archlinux packages are supported!
 
-Blacklist is controled by 3 systemd services:
+Blacklist is controlled by 3 system services:
```
Comment (hugovk) [inline README.md:20]: systemd is correct, it's a daemon

**Comment by Salamek** [comment]
@hugovk @vishalvishw10 Umm did you guys just used my project as a example on some Python/"How to use GitHub" workshop? i just had like 30 pull requests ?from india? with bunch of garbage to close...  only like 3 PR's made sense including this one...

**Comment by hugovk** [comment]
No, it's happening all across GitHub, spammers want a free t-shirt... :(

See:

* https://blog.domenic.me/hacktoberfest/
* https://twitter.com/shitoberfest

I suggest you close them, mark as `invalid` so they don't count towards Hacktoberfest, and block and report them on GitHub.

And consider to "Limit to prior contributors", but remember to remove on 1st November.

https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/limiting-interactions-in-your-repository

---

This PR is one of the least worst, it's actually correcting instead of breaking stuff.

**Comment by Salamek** [comment]
@hugovk thx for the info... i marked them as invalid... but i dont have time to report them all correctly (i would have to keep a list of usernames lol -> not worth of my time ATM)

**Comment by Salamek** [comment]
aaand also added a `spam` label to them...

**Pull request closed without merging**